### PR TITLE
Add Skiff Mail

### PR DESCRIPTION
**Pull request by jonaharagon**
Changes proposed in this PR:

- Recommend Skiff Mail

~~Obviously they don't meet our criteria yet, so this is waiting on the changes from https://discuss.privacyguides.net/t/skiff-mail-email-provider/11411/111, but I think those will eventually be made so I'll get started on writing this now.~~

- [x] I have disclosed any relevant conflicts of interest in my post.
- [x] I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
- [x] I am the sole author of this work. 
- [x] I agree to the [Community Code of Conduct](https://www.privacyguides.org/en/code_of_conduct/).

### CODE REVIEW

**Comment by amilich** [comment]
Thanks so much for opening this! Couple comments if this is helpful:
```
You can create up to 3 additional @skiff.com email aliases in addition to your primary account address on their free plan. [Custom domains](https://skiff.com/blog/custom-domain-setup) are available on their Pro or Business plan, and allow you to create unlimited aliases.
```
May want to do "Essential, Pro, or Business Plan" - the Essential plan was just [rolled out](https://skiff.com/pricing)

I noticed this section in StartMail:
> StartMail allows for proxying of images within emails. If you allow the remote image to be loaded, the sender won't know what your IP address is.

Skiff also does this, which could be an important feature given that it does seem quite standard among private email providers.

**Comment by jonaharagon** [comment]
@amilich yeah, I haven't filled in all of those sections yet. I've also had a difficult time finding some policies, for example, do free Skiff accounts expire after a certain period of inactivity, and what happens when a paid account goes unpaid? I assume it reverts to free, but what happens to any paid features that might already be in use?

**Comment by amilich** [comment]
> @amilich yeah, I haven't filled in all of those sections yet. I've also had a difficult time finding some policies, for example, do free Skiff accounts expire after a certain period of inactivity, and what happens when a paid account goes unpaid? I assume it reverts to free, but what happens to any paid features that might already be in use?

No expiration. If a paid account stops paying, they'll see a banner warning them to remove paid features (like extra aliases) or to renew. Happy to share more details.

**Comment by amilich** [comment]
Hello! Seeing the logo commits - anything I can do to help?

**Comment by amilich** [comment]
> @amilich yeah, I haven't filled in all of those sections yet. I've also had a difficult time finding some policies, for example, do free Skiff accounts expire after a certain period of inactivity, and what happens when a paid account goes unpaid? I assume it reverts to free, but what happens to any paid features that might already be in use?

Any update at all?

**Comment by amilich** [comment]
> Changes proposed in this PR:
> 
> * Recommend Skiff Mail
> 
> Obviously they don't meet our criteria yet, so this is waiting on the changes from https://discuss.privacyguides.net/t/skiff-mail-email-provider/11411/111, but I think those will eventually be made so I'll get started on writing this now.

Hey all - just noting that this is completely resolved. We've also added `skiff-crypto` to the Skiff Mail repo, added additional pages to our site - https://skiff.com/transparency and https://skiff.com/about-us - and more. 

Is there any way to have a more focused discussion on what you're looking for? Or, can we get this PR in?

File: docs/email.md [hunk 1]
```diff
@@ -208,6 +207,50 @@ StartMail allows for proxying of images within emails. If you allow the remote i
...
+
+You can create up to 3 additional @skiff.com email aliases in addition to your primary account address on their free plan. [Custom domains](https://skiff.com/blog/custom-domain-setup) are available on their Pro or Business plan, and allow you to create unlimited aliases.
+
+#### :material-alert-outline:{ .pg-orange } Private Payment Methods
+
+Skiff accepts cryptocurrency payments via Coinbase Commerce, including Bitcoin and Ethereum, but they do not accept our recommended [cryptocurrency](cryptocurrency.md) Monero. They also accept credit card payments via Stripe.
+
+#### :material-check:{ .pg-green } Account Security
+
+Skiff Mail supports TOTP two-factor authentication. They do not allow FIDO2 security key authentication.
```
Comment (amilich) [inline docs/email.md:238]: nit - we did release FIDO2 auth (since this PR was opened :)

**Review by freddy-m — APPROVED** [review]

File: docs/email.md [hunk 2]
```diff
@@ -152,14 +151,65 @@ Mailbox.org has a digital legacy feature for all plans. You can choose whether y
...
+
+!!! recommendation
+
+    ![Skiff Mail logo](assets/img/email/skiff-mail.svg){ align=right }
+
+    **Skiff Mail** is a web based email service with E2EE that began in 2020 that is based in San Francisco with developers worldwide. Accounts start with 10GB of free storage.
+
+    [:octicons-home-16: Homepage](https://skiff.com/mail){ .md-button .md-button--primary }
+    [:octicons-eye-16:](https://app.skiff.com/docs/db93c237-84c2-4b2b-9588-19a7cd2cd45a#tyGksN9rkqbo2uGYASxsA6HVLjUoly/wTYK8tncTto8=){ .card-link title="Privacy Policy" }
+    [:octicons-info-16:](https://skiff.com/help){ .card-link title=Documentation}
```
Comment (mfwmyfacewhen) [inline docs/email.md:173]:
```suggestion
    [:octicons-home-16: Homepage](https://skiff.com/mail){ .md-button .md-button--primary }
    [:octicons-eye-16:](https://app.skiff.com/docs/db93c237-84c2-4b2b-9588-19a7cd2cd45a#tyGksN9rkqbo2uGYASxsA6HVLjUoly/wTYK8tncTto8=){ .card-link title="Privacy Policy" }
    [:octicons-info-16:](https://skiff.com/help){ .card-link title=Documentation}
    [:octicons-code-16:](https://github.com/skiff-org/skiff-apps){ .card-link title="Source Code" }
```

File: docs/email.md [hunk 3]
```diff
@@ -152,14 +151,65 @@ Mailbox.org has a digital legacy feature for all plans. You can choose whether y
...
+
+Skiff has undergone a few [audits](https://skiff.com/transparency) during its development.
+
+#### :material-check:{ .pg-green } Custom Domains and Aliases
+
+You can create up to 3 additional @skiff.com email aliases in addition to your primary account address on their free plan. [Custom domains](https://skiff.com/blog/custom-domain-setup) are available on their Pro or Business plan, and allow you to create unlimited aliases.
+
+#### :material-alert-outline:{ .pg-orange } Private Payment Methods
+
+Skiff Mail accepts cryptocurrency payments via Coinbase Commerce, including Bitcoin and Ethereum, but they do not accept our recommended [cryptocurrency](cryptocurrency.md) Monero. They also accept credit card payments via Stripe.
```
Comment (mfwmyfacewhen) [inline docs/email.md:189]:
```suggestion
Skiff Mail accepts cryptocurrency payments via Coinbase Commerce, including Bitcoin and Ethereum, but they do not accept our recommended [cryptocurrency](cryptocurrency.md), Monero. They also accept credit card payments via Stripe.
```

File: docs/email.md [hunk 4]
```diff
@@ -152,14 +151,65 @@ Mailbox.org has a digital legacy feature for all plans. You can choose whether y
...
+
+#### :material-check:{ .pg-green } Account Security
+
+Skiff Mail supports TOTP two factor authentication and hardware security keys using FIDO2 or U2F standards. The use of a hardware security key requires setting up TOTP two factor authentication first.
```
Comment (mfwmyfacewhen) [inline docs/email.md:193]:
```suggestion
Skiff Mail supports TOTP two-factor authentication and hardware security keys using FIDO2 or U2F standards. The use of a hardware security key requires setting up TOTP two-factor authentication first.
```

**Review by mfwmyfacewhen — CHANGES_REQUESTED** [review]

**Comment by amilich** [comment]
@jonaharagon really appreciate all the work getting Skiff Mail close!

Anything I can do to help? Happy to make the small changes from @mfwmyfacewhen

**Review by blacklight447 — APPROVED** [review]
Lgtm

**Review by mfwmyfacewhen — APPROVED** [review]

**Pull request merged**